examples/edit.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+#include <cstdlib>
+#include <string>
+
+#include "vega/dictionary/dictionary.h"
+#include "vega/dicom/file.h"
+
+int main(int argc, char *argv[]) {
+  if (argc < 3){
```

Comment:
I don't link to any specific style guide, but internally I have been trying to follow the Google C++ style guide.

https://google.github.io/styleguide/cppguide.html#Conditionals:
"put one space between the if and the opening parenthesis, and between the closing parenthesis and the curly brace"